### ISSUE TITLE
Strip original assets from game files

### ISSUE THREAD

**Issue by therationalpi** [Backdash/MonikaModDev#42]
Team Salvato has posted their IP Guidelines, including the stance on fan games and mods (like ours). I think it's very important that we be respectful of their wishes, especially considering how fair and open these guidelines are.

With that in mind, there are two points we really need to address to meet their guidelines.

> Any mods must NOT be distributed as a complete game. They should contain only the files that are necessary to install the mod (usually files that are added to the DDLC game folder). Most mods only require the scripts.rpa file to be replaced – sometimes images.rpa if new graphics have been added to the game. In this example, please distribute your mod ONLY as these files, so that the user installs it into their existing official DDLC game.

Currently we not only include files that make After Story a standalone game, but we also host a full repository of original assets in unedited and un-obfuscated form. In order to meet their guidelines, we'll need to adjust the build settings for Renpy, and also delete a lot of files from the repo. Once this is done, **all contributors will be required to rebase before submitting new pull requests from remote forks**. This is necessary to ensure that original assets do not end up being replaced in our repo.

On the plus side, the resulting repo will be much cleaner and easier to navigate, and the download size for the mod will be very very small. A project to do exactly this has been on under consideration for a while, and the change was likely to happen in version 0.5.0 regardless of Team Salvato's recent post.

*In case anyone is bothered by Team Salvato's position on this. While you are entitled to your opinion, I personally feel that the guidelines put forward are very fair and reasonable (generous, even), and reflect that Dan has been a mod creator before.*

The other point is with regards to disclaimers, and is in another issue.

**Comment by therationalpi** [comment]
After contacting Dan directly, he seems to be fine with us hosting assets on the repo as a reference for future builds. Assets still must not be included in future releases, but scrubbing the repo's history is unnecessary.

If Team Salvato changes their mind on this in the future, we'll revisit the issue.

**Comment by therationalpi** [comment]
A branch for this is now live.

https://github.com/Backdash/MonikaModDev/tree/lightweight

This branch is based on the work for the [DDLC Mod Template](https://github.com/therationalpi/DDLCModTemplate), and migration will consist of moving individual files from the current master to lightweight as needed.

**Comment by Jxhnathan08** [comment]
Due to the lightweight nature of the template, a significant amount of files will either be kept in .rpa files or left out entirely. This will cut down on clutter & aid with organization in the future.
